include/wx/apptrait.h

```diff
@@ -209,13 +211,16 @@ class WXDLLIMPEXP_BASE wxConsoleAppTraitsBase : public wxAppTraits
     virtual bool HasStderr() wxOVERRIDE;
 
     // the GetToolkitVersion for console application is always the same
-    virtual wxPortId GetToolkitVersion(int *verMaj = NULL, int *verMin = NULL) const wxOVERRIDE
+    wxPortId GetToolkitVersion(int *verMaj = NULL,
```

Comment:
We still seem to use `virtual` even together with `wxOVERRIDE` in other places, so we probably should keep it here too.